projects/packages/identity-crisis/changelog/update-remove-identity-crisis-deprecated-code

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Removing deprecated code should be `major`.

In this case it doesn't really matter since the package is still in 0.x versions, but keeping an eye out for this sort of thing is good practice. 🙂